src/main/java/net/p3pp3rf1y/sophisticatedbackpacks/command/DynamicCommand.java

```diff
@@ -0,0 +1,235 @@
+package net.p3pp3rf1y.sophisticatedbackpacks.command;
+
+import com.google.common.cache.Cache;
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.RemovalNotification;
+import com.mojang.brigadier.arguments.IntegerArgumentType;
+import com.mojang.brigadier.arguments.StringArgumentType;
+import com.mojang.brigadier.builder.ArgumentBuilder;
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
+import net.minecraft.commands.CommandBuildContext;
+import net.minecraft.commands.CommandSourceStack;
+import net.minecraft.commands.Commands;
+import net.minecraft.commands.arguments.item.ItemArgument;
+import net.minecraft.commands.arguments.item.ItemInput;
+import net.minecraft.core.registries.BuiltInRegistries;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.server.level.ServerLevel;
+import net.minecraft.server.level.ServerPlayer;
+import net.minecraft.sounds.SoundEvents;
+import net.minecraft.sounds.SoundSource;
+import net.minecraft.world.entity.item.ItemEntity;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.phys.Vec3;
+import net.neoforged.neoforge.items.ItemStackHandler;
+import net.p3pp3rf1y.sophisticatedbackpacks.backpack.BackpackTemplates;
+import net.p3pp3rf1y.sophisticatedbackpacks.backpack.wrapper.BackpackWrapper;
+import net.p3pp3rf1y.sophisticatedbackpacks.backpack.wrapper.IBackpackWrapper;
+import net.p3pp3rf1y.sophisticatedcore.util.InventoryHelper;
+import net.p3pp3rf1y.sophisticatedcore.util.RandHelper;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+public class DynamicCommand {
+	static final Cache<String, DynamicTemplate> DYNAMIC_CACHE = CacheBuilder.newBuilder().removalListener(DynamicCommand::onRemoval).expireAfterAccess(5, TimeUnit.MINUTES).build();
+
+	static ArgumentBuilder<CommandSourceStack, ?> register(CommandBuildContext commandBuildContext) {
+		return Commands.literal("dynamic")
```

Comment:
I believe this should be under template command as suggested in your comment or it needs better name like `dynamic_template` so that it's clearer, but am really leaning to it being under template.